src/main/javascript/email_settings.js

```diff
@@ -276,6 +283,7 @@ const EmailSettingsTableManager = ({ fleetUsers }) => {
             url: '/protected/update_email_preferences',
             data: submissionData,
             dataType: 'json',
+            async: false,
```

Comment:
So just make sure the async flags are set intentionally. These freeze the web page so we don't want to do that unless its necessary. Usually you would disable some elements on the web page until the contingent request returns rather than using a synchronous call.